ui/src/data-products/translations.fi.json

```diff
@@ -51,6 +51,13 @@
                 "BIQUADRATIC_PARABOLA": "Siirtymäkaari (Helmert)",
                 "MISSING_SECTION": "Linkitys puuttuu"
             }
+        },
+        "vertical-geometry": {
+            "vertical-geometry-title": "Pystygeometria",
+            "location-track-vertical-geometry": "Raiteen pystygeometria",
+            "plan-vertical-geometry": "Suunnitelman pystygeometria",
+            "location-track-search-legend": "Raiteen pystygeometria näyttää luettelossa paikannuspohjan raiteeseen liityvät taitepisteet. Paikannuspohjan raide voi olla koostettu useasta suunnitelmasta, eikä täysin vastaa sitä suunnitelmakokonaisuutta, jonka mukaan raide on rakennettu, joten luettelon tietoja on syytä pitää epäluotettavina. Luettelon tietoja voi käyttää esim. kokonaiskuvan hahmottamiseen tai ongelmien selvittämiseen, mutta niitä ei pidä käyttää esim. tukemiskoneen työn lähtötietona.",
+            "plan-search-legend": "Suunnitelman pystygeometria näyttää luettelossa suunnitelman sisältämät taitepisteet. Rataosoiteet on laskettu paikannuspohjan pituusmittauksen mukaan, joten rataosoitteiden tarkkuutta on syytä pitää epäluotettavana. Myös  paikannuspalvelun suunnitelmien elementtien tietoja on syytä pitää epäluotettavina, joten niitä ei pidä käyttää esim. tukemiskoneen työn lähtötietona."
```

Comment:
Tuossa "Myös  paikannuspalvelun" -kohdassa taitaa olla kaksi välilyöntiä peräkkäin